WalletWasabi/Services/WalletService.cs

```diff
@@ -253,6 +253,16 @@ private async void IndexDownloader_NewFilterAsync(object sender, FilterModel fil
 			}
 		}
 
+		public static void DropWalletCache(string walletName, string dataDir, Network network)
+		{
+			string FiltersFolderPath = Path.Combine(dataDir, "IndexStore", network.ToString());
```

Comment:
`FiltersFolderPath` should be camelCase.

Reply:
I do, I think this might have been a result of me copy-pasting.